plugins/org.eclipse.n4js.transpiler.es/src/org/eclipse/n4js/transpiler/es/transform/ModuleWrappingTransformation.xtend

```diff
@@ -407,16 +405,16 @@ class ModuleWrappingTransformation extends Transformation {
 		val Iterable<VariableDeclaration> allHoisted = scriptContent.map[ scriptElement |
 			val Iterable<VariableDeclaration> ret = switch( scriptElement ) {
 				VariableStatement : {
-					val pairsOfDeclarationAndInitialsers = scriptElement.varDecl.toHoistDeclaration;
+					val pairsOfDeclarationAndInitialsers = scriptElement.varDeclsOrBindings.toHoistDeclarations;
 					val listOfInitExpressionStmts = pairsOfDeclarationAndInitialsers.map[it.value].filterNull.toList;
 					if( !listOfInitExpressionStmts.empty ) mapVarStatement2replacer.put( scriptElement, listOfInitExpressionStmts );
-					pairsOfDeclarationAndInitialsers.map[it.key]
+					pairsOfDeclarationAndInitialsers.map[it.key].flatten
```

Comment:
Here, I just kept the old code unchanged.
A small advantage of `it.key` is that the risk of shadowing issues is smaller.